pymatgen/analysis/chemenv/utils/graph_utils.py

```diff
@@ -268,7 +268,7 @@ def from_edges(cls, edges, edges_are_ordered: bool = True) -> Self:
         order in the list.
         """
         if edges_are_ordered:
-            nodes = [e[0] for e in edges]
```

Comment:
Like this, this may or may not be necessary. I don't think people would be confused by this.